### PR TITLE
stakeStart() native

### PR DESCRIPTION
**Pull request by i-skrypnyk**
## Description
stakeStart native method

## Jira Ticket
[SDK-1804](https://horizenlabs.atlassian.net/browse/SDK-1804)

[SDK-1804]: https://horizenlabs.atlassian.net/browse/SDK-1804?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: qa/sc_evm_native_forger_v2.py [hunk 1]
```diff
@@ -595,6 +608,22 @@ def run_test(self):
...
+                                                               forger_1_vrf_pub_key_to_bytes[32:],
+                                                               "0x" + evm_address_sc_node_1)
+        result = sc_node_1.rpc_eth_call(
+            {
+                "to": "0x" + FORGER_STAKE_V2_SMART_CONTRACT_ADDRESS,
+                "from": add_0x_prefix(evm_address_sc_node_1),
+                "input": data_input
+            }, "latest"
+        )
+        assert_equal(decode(['uint32'], hex_str_to_bytes(result['result'][2:]))[0], VERSION_1_4_FORK_EPOCH)
```
Comment (alsala) [inline qa/sc_evm_native_forger_v2.py:625]: It works here, but actually it should be decoded as an an `int32`. Moreover maybe we could add a test using an address which did not delegate anything and checking that it returns `-1`.

File: qa/sc_evm_native_forger_v2.py [hunk 2]
```diff
@@ -595,6 +608,22 @@ def run_test(self):
...
+
```
Comment (alsala) [inline qa/sc_evm_native_forger_v2.py:626]: Another interesting test would be to request the epoch for genesis staking address and see that it returns 1

**Review by saratnt — APPROVED** [review]

File: qa/sc_evm_native_forger_v2.py (hunk 1 above)
Comment (i-skrypnyk) [reply qa/sc_evm_native_forger_v2.py]: Thanks, fixed and added a check

File: qa/sc_evm_native_forger_v2.py (hunk 2 above)
Comment (i-skrypnyk) [reply qa/sc_evm_native_forger_v2.py:641]: I will cover this under https://horizenlabs.atlassian.net/browse/EON-1826 as I am adding more fo.rgers with different config there

**Pull request merged**